README.md

```diff
@@ -1,17 +1,79 @@
+| Client SDK    | Refers to SDKs used by Web, Flutter, Android, and Apple applications. Use as a proper noun, use "a Client SDK" or "the Client SDKs", not client-side SDKs.                                                        |
+| Server SDK    | Refers to SDKs used by backend languages like Java, Node.js, or PHP. Use as a proper noun, use "a Server SDK" or "the Server SDKs", not server-side SDKs.                                                         |
+| Developer     | Refers to Appwrite developers that use Appwrite to create applications.                                                                                                                                           |
+| End User      | Refers to end users of applications with an Appwrite backend. This doesn't refer to developers that interact with Appwrite directly.   |
+| Appwrite Instance | Refers to a single, self-hosted deployment of Appwrite   |
+| Application   | Refers to the application built by the Appwrite developer. Can be referred to as web app, mobile app, flutter app, etc. |
+| Method                | Refers to SDK methods like the method `account.get()`. This helps differentiate SDK methods and Appwite Functions in written language.                                                                                                                             |
+| Parameters            | Refers to parameters in an SDK method or API definition. Use in cases where we speak about needs, such as "takes parameters x, y, and z" or "requires parameters x and y".                                                                                         |
+| Arguments             | Refers to arguments passed to CLI, SDK method, or REST API. Use when the argument has a concrete value and when we refer to invoking a method, such as "pass in arguments x and y, or "call account.create() with the argument "unique()" to generate a random id. |
 
-#### Spell Checking
+#### Authentication
+| **Term**               | **Suggested Usage.**                                                                                                                                                                                                                                          |
+|------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Authentication Service | The Authentication service encompases the Account API, Users API, and Teams API.                                                                                                                                                                              |
```

Comment:
### Need Feedback

Do we want to call services
1. Appwrite Authentication
2. Authentication Service (entire proper noun, which is proper if we're consistent)
3. Authentication service (where Authentication, the service, is the sole proper noun)